lib/expansion_rules.rb

```diff
@@ -66,7 +66,17 @@ def details_fields(*fields)
   TRAVEL_ADVICE_FIELDS = (DEFAULT_FIELDS + details_fields(:country, :change_description)).freeze
   WORLD_LOCATION_FIELDS = [:content_id, :title, :schema_name, :locale, :analytics_identifier].freeze
   FACET_GROUP_FIELDS = (%i[content_id title schema_name] + details_fields(:name, :description)).freeze
-  FACET_FIELDS = (%i[content_id title schema_name] + details_fields(:key)).freeze
+  FACET_DETAILS_FIELDS = %i[
```

Comment:
This feels like an unnecessary constant definition as it's not used anywhere else.